### DISCUSSION TITLE
Weread Dataview | 让彩虹猫咪来追踪你的微信读书进度吧■■■■■■□□□□□□

### DISCUSSION THREAD

**Discussion by WinHex89** [zhaohongxuan/obsidian-weread-plugin#288] [Ideas]
![小猫文章配图1](https://github.com/zhaohongxuan/obsidian-weread-plugin/assets/134070684/8b3c2bd3-b3d1-4b93-b51a-0b694375e2bd)

**Dataview+Weread**✨

👉[喂饭教程文章链接](https://mp.weixin.qq.com/s?__biz=Mzg3NTk4Mjg5NA==&mid=2247484261&idx=1&sn=3a4b529c1886688b1bbcce09ae79cd02&chksm=cf387a24f84ff3326d4acdee65edb919a00edb975d64c44e585a3f74783bd28351e694d05f8e&token=109013616&lang=zh_CN#rd)

![效果展示1](https://github.com/zhaohongxuan/obsidian-weread-plugin/assets/134070684/d0a7a84f-bb10-4511-9d09-57c675749efe)

**Comment by ZiGmaX809** [comment]
用上了  还加了Badges

**Comment by ZiGmaX809** [comment]
> 用上了 还加了Badges

发现ios上面无法使用::after伪元素，所以用<div>方式重新实现了一下以支持移动端

[nyancat-div](https://raw.githubusercontent.com/ZiGmaX809/PrivateRules/master/Others/nyancat-div.css)

引用方式：

```html
<div class="nyan-cat-progress-container"><div class="nyan-cat-progress-bar" value=8><div class="nyan-cat-rainbow"><div class="nyan-cat-img"></div></div></div></div>
```
